src/forms/forms.controller.ts

```diff
@@ -19,7 +19,7 @@ import {
 import { FormsService } from './forms.service';
 
 @ApiTags('forms')
-@ApiBearerAuth('access-token')
+// @ApiBearerAuth('access-token')
```

Comment:
Remove?

Reply:
This decorator is now being applied individually to each route, using the custom decorator created by Vinicius. When authRoute is true, it applies @ApiBearerAuth('access-token') to the route in question. In this case, to maintain the same logic as before, I believe authRoute should remain true for all the form endpoints.